database/migrations/2022_07_08_085310_add_allow_performance_to_osu_builds.php

```diff
@@ -30,6 +31,7 @@ public function down()
     {
         Schema::table('osu_builds', function (Blueprint $table) {
             $table->dropColumn('allow_performance');
+            $table->dropIndex('allow_performance');
```

Comment:
Not sure if order matters but this feels backwards 🤔 
I guess laravel is smart enough to run correctly?

Reply:
:| I looked at other migrations in reference to this, but yes this does indeed fail. As should e.g. https://github.com/ppy/osu-web/blob/2ffc17cc6268898f1f9e19aa304436fcbaa9490c/database/migrations/2021_08_30_153214_add_track_id_to_beatmapsets.php#L30-L36

Reply:
Good thing we don't rollback migrations.. ever 🙈